<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tke_Form_uni_insert_subs</class>
 <widget class="QWidget" name="tke_Form_uni_insert_subs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="tke/tke_rc">
    <normaloff>:/pictures/pictures/cube.png</normaloff>:/pictures/pictures/cube.png</iconset>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_chooseFile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Вибір файлу субсидії</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeView" name="treeView_fileMeneger"/>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_reloadDirTree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Поновити дерево каталогів</string>
        </property>
        <property name="icon">
         <iconset resource="tke/tke_rc">
          <normaloff>:/pictures/pictures/redeal.png</normaloff>:/pictures/pictures/redeal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_reloadScript">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Поновити скрипт</string>
        </property>
        <property name="icon">
         <iconset resource="tke/tke_rc">
          <normaloff>:/pictures/pictures/redeal.png</normaloff>:/pictures/pictures/redeal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_subs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Субсидії з файлу</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget_subsydii">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>status</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Рахунок</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Сума ЦО, грн.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Донарах ЦО, грн.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Сума ГВ, грн.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Донарах ГВ, грн.</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_SubsSum">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_SubsSum_GV">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label_date">
          <property name="text">
           <string>Місяць</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_date">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximumDate">
           <date>
            <year>2100</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
          <property name="minimumDate">
           <date>
            <year>2000</year>
            <month>9</month>
            <day>14</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>MM.yyyy</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_addSubs">
          <property name="text">
           <string>Додати стрічку</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/greplus.png</normaloff>:/pictures/pictures/greplus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_dellSubs">
          <property name="text">
           <string>Видалити стрічку</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/redminus.png</normaloff>:/pictures/pictures/redminus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_insertSubsToDB">
          <property name="text">
           <string>Внести субсидії до бази даних</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/ark.png</normaloff>:/pictures/pictures/ark.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tke/tke_rc"/>
 </resources>
 <connections/>
</ui>
